.cargo/config

```diff
@@ -2,3 +2,4 @@
 wasm = "build --release --target wasm32-unknown-unknown --lib"
 wasm-debug = "build --target wasm32-unknown-unknown"
 unit-test = "test --lib"
+integration-test = "test --package e2e -- --ignored --test-threads 1 -Z unstable-options --report-time"
```

Comment:
https://doc.rust-lang.org/beta/unstable-book/compiler-flags/report-time.html


This allows us to see the runtime of each integration test.